src/components/policy/policy_regular/src/cache_manager.cc

```diff
@@ -585,6 +585,13 @@ bool CacheManager::SecondsBetweenRetries(std::vector<int>& seconds) {
 
   const std::size_t size =
       pt_->policy_table.module_config.seconds_between_retries.size();
+
+  const std::size_t max_seconds_between_retries_size = 5u;
+  if (size > max_seconds_between_retries_size) {
+    // must exit app
+    exit(EXIT_FAILURE);
```

Comment:
@pold500 I suppose that it is very bad idea to immediately exit application on that case. 
First because this function can be called not only on startup.
Second because you didn't log anything. Just imagine that SDL stops immediately after Start. How would you find issue why it is dound it?